### ISSUE TITLE
potential copyright issue from transitive dependency "wordwrap"

### ISSUE THREAD

**Issue by beiwei30** [eslint/eslint#11536]
[wordwrap](https://github.com/substack/node-wordwrap) publishes its test directory in which a text file 'idleness.txt' is included. This may introduce potential copyright issue.

see more from https://github.com/substack/node-wordwrap/issues/21

**Comment by aladdin-add** [comment]
hi, thanks for creating the issue!
is `wordwrap` eslint's dependency(I'm not seeing it in package.json)?

**Comment by beiwei30** [comment]
here's the dependency tree:

```
├─┬ eslint@3.19.0
│ └─┬ optionator@0.8.2
│   └── wordwrap@1.0.0
```

**Comment by beiwei30** [comment]
here's the dependency tree which eventually introduce the file in question.

```
└─┬ webpack@3.12.0
  └─┬ uglifyjs-webpack-plugin@0.4.6
    └─┬ uglify-js@2.8.29
      └─┬ yargs@3.10.0
        └─┬ cliui@2.1.0
          └── wordwrap@0.0.2
```

**Comment by nzakas** [comment]
I’ve contacted the Open JS Foundation to get some insights into whether or not there is a copyright issue here. Thanks for the report, I will update when I hear back.

**Comment by nzakas** [comment]
Given that this has been an issue for four years, there's probably not a huge urgency here. However, we should work with Optionator to see if it would be possible to switch to another dependency.

Related issue: https://github.com/gkz/optionator/issues/31

**Comment by ilyavolodin** [comment]
Unfortunately, it seems like optionator is dead at this point. I filed an issue with them a few month back, and never seen any response to it.

**Comment by kaicataldo** [comment]
Do we want to be looking for alternatives to optionator?

**Comment by ilyavolodin** [comment]
I looked at some of the available packages that deal in the same space, but most of them are very free formed. As in, they will parse whatever is given to them, not enforce specific constrains in terms of type and  number of value.

**Comment by gkz** [comment]
I can look into fixing this guys

**Comment by gkz** [comment]
Feel free to email me at the email in my profile if you have such issues in the future

**Comment by kaicataldo** [comment]
@gkz Thank you!

**Comment by gkz** [comment]
Do you guys have a suggestion for an alternative to using `wordwrap`? It seems unlikely to be updated

**Comment by kaicataldo** [comment]
I'm seeing a number of packages that look similar to `wordwrap` [on npm](https://www.npmjs.com/search?q=word%20wrap), though I don't have experience with any of them. Since the project is MIT licensed, another option could potentially be to copy [the code](https://github.com/substack/node-wordwrap/blob/master/index.js) (since it's a pretty small module) and its tests into `optionator` and, if the author of `wordwrap` ever updates it, switch back.

**Comment by kaicataldo** [comment]
Friendly ping - is this still an issue?

**Comment by nzakas** [comment]
@gkz have you been able to replace wordwrap in optionator?

**Comment by gkz** [comment]
I've updated the dependency to `word-wrap` instead of `wordwrap`. Let me look into upgrading other dependencies and then I'll release a new version. 

https://github.com/gkz/optionator/commit/bf4b1da28ba085646e0ae9bae602542176f8e635

**Comment by gkz** [comment]
I published Optionator 0.8.3 with this change, let me know if there are any issues.

**Comment by kaicataldo** [comment]
Thanks for the update!
